packages/core/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "lib": ["esnext"],
     "paths": {
       "~/*": ["src/*"]
-    }
+    },
+    "noUnusedLocals": true
```

Comment:
let's remove this option while the project is still in early stages

Reply:
Done